akka-http/src/main/scala/akka/http/scaladsl/server/directives/PathDirectives.scala

```diff
@@ -143,8 +146,8 @@ trait PathDirectives extends PathMatchers with ImplicitPathMatcherConstruction w
    * }}}
    *
    * For further information, refer to:
-   * @see [[http://googlewebmastercentral.blogspot.de/2010/04/to-slash-or-not-to-slash.html]]
    *
+   * @see [[http://googlewebmastercentral.blogspot.de/2010/04/to-slash-or-not-to-slash.html]]
```

Comment:
Would you mind to fix this to the redirected location? https://webmasters.googleblog.com/2010/04/to-slash-or-not-to-slash.html